### PR TITLE
Implement Data for chrono types

### PR DESCRIPTION
**Pull request by r-ml**
Closes https://github.com/linebender/druid/issues/1736

### CODE REVIEW

**Comment by r-ml** [comment]
Question: Do we re-export like with the `im` crate?

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ You can find its changes [documented below](#070---2021-01-01).
   context-methods to implement disabled ([#1632] by [@xarvic])
 - `LifeCycle::BuildFocusChain` to update the focus-chain ([#1632] by [@xarvic])
 - `DisabledIf` widget wrapper to disable based on the state of Data and Env ([#1702] by [@xarvic])
+- `chrono` feature with `Data` support for [chrono](https://docs.rs/chrono/) types ([#1743] by [@r-ml])
```
Comment (cmyr) [inline CHANGELOG.md:37]: you'll also need to add the links to `[#1743]` and `[@r-ml]`, below.

**Review by cmyr — APPROVED** [review]
Thanks! I think there are a few things missing from the changelog, but otherwise this looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (r-ml) [reply CHANGELOG.md:37]: I think I did it correctly

Comment (cmyr) [reply CHANGELOG.md:37]: yep looks good!

**Pull request merged**